torch/csrc/generic/methods/Tensor.cwrap

```diff
@@ -824,3 +824,38 @@ PyObject * THPTensor_(copy_)(PyObject *self, PyObject *args, PyObject *kwargs)
   return THPCopyMethod(THTensor_(copy_functions), self, args, kwargs);
   END_HANDLE_TH_ERRORS
 }
+
+[[
+  name: bitand
```

Comment:
Would it work if you used `name: __and__`? I think you could remove the python function in that case.